src/main/java/com/google/devtools/build/lib/skyframe/BazelSkyframeExecutorConstants.java

```diff
@@ -40,7 +42,14 @@ private BazelSkyframeExecutorConstants() {}
    * therefore fails the build, this ignore functionality currently has no chance to kick in.
    */
   public static final SkyFunction IGNORED_PACKAGE_PREFIXES_FUNCTION =
-      new IgnoredPackagePrefixesFunction(PathFragment.create(".bazelignore"));
+      new IgnoredPackagePrefixesFunction(BAZELIGNORE_PATH, true);
```

Comment:
IMO it would be better to merge these two args -- or even better, for the NOOP version, just have an inline SkyFunction that returns an empty object. So the non-NOOP version doesn't need any ctor args. `BAZELIGNORE_PATH` could even just be moved to IgnorePackagePrefixesFunction.

Reply:
Good idea. Done. This made `IgnoredPackagePrefixesFunction` slightly nicer.